tests/integration/Sync_test.cpp

```diff
@@ -951,12 +951,8 @@ struct StandardClient : public MegaApp
             return false;
         }
 
-        string localpath;
-        n->getlocalpath(&localpath, false);
-        ::mega::FileSystemType fileSystemType = client.fsaccess->getFilesystemType(&localpath);
-        client.fsaccess->local2name(&localpath, fileSystemType);
-        string n_localname = n->localname;
-        client.fsaccess->local2name(&n_localname, fileSystemType);
+        auto localpath = n->getLocalPath(false).toName(*client.fsaccess, FS_UNKNOWN);
+        string n_localname = n->localname.toName(*client.fsaccess, FS_UNKNOWN);
```

Comment:
why removing getting file system type from local path? :thinking:

Reply:
Ah yes - the filenames used in these tests are all simple ascii so no character conversion is needed - also, to make that call we would need a utf8 string to pass, which we don't have readily available, only the LocalPath (which for windows is utf16 of course).   So this is actually one of those cases you suspected, where we would not be getting the correct file system type anyway (on windows), due to inquiring about the wrong sort of string.   But since it won't make any difference for this test, it's easier just to pass the unknown value.